source/manual/data-sources-for-transition.html.md

```diff
@@ -0,0 +1,26 @@
+---
+owner_slack: '#navigation'
+review_by: 2017-07-26
+title: Data Sources for Transition
+section: Transition
+layout: manual_layout
+parent: "/manual.html"
+old_path_in_opsmanual: "../opsmanual/2nd-line/applications/bouncer-and-transition.html"
+---
+
+# Data sources for Transition
+
+Site configuration is automatically imported on an hourly basis via
+[deploy.publishing](https://deploy.publishing.service.gov.uk/job/Transition_load_site_config/) from
+[transition-config](https://github.com/alphagov/transition-config).
+
+Pre-transition traffic data is imported from
+[pre-transition-stats](https://github.com/alphagov/pre-transition-stats),
+based on logs provided by transitioning organisations. This was updated
```

Comment:
Importing data for sites before they transition happens on a one-off basis - if we decide we need to do this for a particular site, we'd get logfiles from the org, process them manually, commit them to this repo and they would get imported - that only needs to happen once.

We did this for local.direct.gov.uk last year, and discussed doing it for at least one of the upcoming PHE transitions, and it should be considered as an option for other future ones, even if we only do it rarely.